projects/grpc-httpjson-transcoding/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# This project uses bazel rules_fuzzing. Script copied from OSS fuzz's bazel-rules-fuzzing-test.
```

Comment:
Like @inferno-chromium said you could replace this with 
https://github.com/google/oss-fuzz/blob/008c7ec1f548755f2e97d5c0ed7e163a42ad32dd/infra/base-images/base-builder/bazel_build_fuzz_tests now!

Reply:
Done, thanks!